src/test/kotlin/com/zepben/evolve/streaming/get/NetworkConsumerClientTest.kt

```diff
@@ -543,6 +584,20 @@ internal class NetworkConsumerClientTest {
         assertThat(consumerClient.getTerminalsForConnectivityNode(connectivityNode), equalTo(expectedResult))
     }
 
+    @Test
+    internal fun `iterable mrids variant coverage`() {
```

Comment:
This one?

Reply:
yep, that is the one, and it now seems the PR is indicating the function is covered. Not sure what went wrong with it.